src/aarch64/ioctl.rs

```diff
@@ -1,1606 +1,3 @@
 /* automatically generated by rust-bindgen 0.66.1 */
 
-pub const FIONREAD: u32 = 21531;
```

Comment:
It looks like a bunch of ioctl codes here are getting removed from src/aarch64/ioctl.rs. Do you know what's causing this?

Reply:
Uhm, makes no sense, the only reasonable explaination is usually that functional macro (which ioctl make use of) are confusing bindgen in some way that I may have hit involuntarily.